src/aktualizr_get/main.cc

```diff
@@ -0,0 +1,77 @@
+#include <unistd.h>
+#include <iostream>
+
+#include <openssl/ssl.h>
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
+
+#include "config/config.h"
+#include "get.h"
+
+#include "utilities/aktualizr_version.h"
+
+namespace bpo = boost::program_options;
+
+bpo::variables_map parse_options(int argc, char *argv[]) {
+  bpo::options_description description("aktualizr-get command line options");
```

Comment:
Might be useful to either print something here or at least have a comment to explain what the tool is for.